plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/GodotExtension.kt

```diff
@@ -13,14 +13,24 @@ open class GodotExtension(objects: ObjectFactory) {
     val singleton = objects.property<Boolean>()
     val loadOnce = objects.property<Boolean>()
     val reloadable = objects.property<Boolean>()
-    val platforms = objects.listProperty<Platform>()
+    internal lateinit var configure: (List<Platform>) -> Unit
 
     internal val entrySourceDir = objects.directoryProperty()
 
     private var configureTargets: KotlinNativeTarget.() -> Unit = {}
 
     fun platforms(vararg platforms: Platform) {
-        this.platforms.set(platforms.toList())
+        configure(platforms.toList())
+    }
+
+    fun defaultPlatforms() {
```

Comment:
This was supposed to be in `1.4-rc` but I needed it here to fix the gradle plugin.